spec/lob/resources/resource_base_spec.rb

```diff
@@ -50,4 +50,46 @@
     subject.states.must_be_kind_of(Lob::Resources::State)
   end
 
+  it "should accept an idempotency_key" do
+    @sample_address_params = {
```

Comment:
We don't need the `@` here, since that is for instance variables and this is just a variable local to this test.